changelog/fragments/ansible-path-issue-fix.yaml

```diff
@@ -0,0 +1,9 @@
+entries:
+  - description: >
+      For Ansible operators, API endpoint has path component in it. If it has path then current flow is ignoring the path component.
+    kind: bugfix
```

Comment:
make this `kind: change`, since it's not really a bugfix yet

Reply:
(that will also appease the docs sanity check)

Reply:
Modified.